build/charts/antrea/templates/crds/externalnode.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This file is moved to path build/charts/antrea/crds, so this is not needed.

Reply:
Rebased the patch on top of feature branch. Pls help with review.